AC-1394 Review Glue API backend docs (#1269)
spryker/spryker-docs: docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/how-to-guides/how-to-create-protected-endpoints.md

---
title: How to create protected endpoints
description: This guide shows how to create the protected endpoint using a resource for the Storefront and Backend API applications.
last_updated: September 30, 2022
template: howto-guide-template
redirect_from:
  - /docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/how-to-guides/create-protected-endpoints.html
---

This guide shows how to create the protected endpoint using a resource for the Storefront and Backend API applications.

> Let's say you have a module named `FooApi`, where you want to have a new protected endpoint `/foo` with `GET` and `POST` methods. For this, follow these steps:

Author: ```suggestion
Let's say you have a module named `FooApi`, where you want to have a new protected endpoint `/foo` with `GET` and `POST` methods.  To create the protected endpoint, follow these steps::
```
